src/components/step/StepText/StepText.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import stepElementStyle from '../step-helpers';
+import './StepText.css'
+import { StepElementProps } from '../shared';
+import StepMarkdown from '../StepMarkdown/StepMarkdown';
+
+
+export default function (props: StepElementProps) {
+    if (!props.text) {
+      return (<></>)
```

Comment:
I assume it's functionally equivalent (render nothing) but I usually just return null